pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools>=64", "setuptools_scm>=8", "cmake", "swig"]
+
+[project]
+name = "pyccl"
+authors = [
+    {name = "LSST DESC"},
+]
+description="Library of validated cosmological functions."
+readme = "README.md"
+requires-python = ">=3.8"
+keywords = ["cosmology", "astrophysics", "physics", "science", "LSST"]
+license = { text = "BSD" }
+dynamic = ["version"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: C",
+    "Programming Language :: Python :: 3",
+    "Topic :: Scientific/Engineering :: Astronomy",
+    "Topic :: Scientific/Engineering :: Physics",
+]
+dependencies = [
+    "numpy",
+]
+
+[project.urls]
+homepage = "https://lsstdesc.org/"
+documentation = "https://ccl.readthedocs.io/en/latest/"
+repository = "https://github.com/LSSTDESC/CCL"
+
+[project.optional-dependencies]
+dev = [
+    "pytest",
+    "pytest-cov",
+]
+
+[tool.setuptools]
+packages = ["pyccl"]
+license-files = ["LICENSE", "LICENSE_COSMICEMU"]
+include-package-data = true
```

Comment:
License files and package data will be included automatically by default and in setup.py we use `find_packages()`, so the four lines can be reduced to the following (to avoid a lot of warnings when `pip install . -v` is issued):

[tool.setuptools.packages.find]
include = ["pyccl*"]